File: Games/NoughtsAndCrosses/src/main/java/jokatu/game/games/noughtsandcrosses/game/Line.java
package jokatu.game.games.noughtsandcrosses.game;

import ophelia.collections.list.UnmodifiableList;

@SuppressWarnings("unused") // This is converted to JSON using Jackson.
public class Line {
	private final String start;
	private final String end;

	Line(UnmodifiableList<Integer> cells) {
		this.start = "cell_" + cells.get(0);
		this.end = "cell_" + cells.get(cells.size() - 1);
	}

	public String getStart() {
		return start;
	}

	public String getEnd() {
		return end;
	}
}

File: Games/NoughtsAndCrosses/src/main/kotlin/jokatu/game/games/noughtsandcrosses/game/Line.kt
package jokatu.game.games.noughtsandcrosses.game

import ophelia.collections.list.UnmodifiableList

// This is converted to JSON using Jackson.
class Line internal constructor(cells: UnmodifiableList<Int>) {
	val start: String = "cell_${cells.get(0)}"
	val end: String = "cell_${cells.get(cells.size() - 1)}"
}
